boards/common/native/doc.md

```diff
@@ -26,6 +26,15 @@ and @ref boards_native64. Using `BOARD=native` will automatically select the rig
 - Network: Tap Interface
 - UART: Runtime configurable - `/dev/tty*` are supported
 - Timers: Host timer
+- RTC: hosttime at 'UTC' to behave consitend with embedded systems if TZ is set that timzone will be used.
+    - set `TZ` enviroment variable e.g.:
+    - execute `export TZ=":/etc/localtime"` prior `make term` in the very shell you intend to run make term in
+    - add `export TZ=":/etc/localtime"` to your `~/.profile`
+    - run `TZ=":/etc/localtime" make term`
+    - set `TZ` in riot application picolibc and newlib are able to respect that setting if `+/-hh:mm:ss`
```

Comment:
Can't we just add this to `pyterm` so it's less user hostile?

Reply:
i will not:  i think doing things wrong (make embedded things that assume no timezone behave wrong) should not be the default. < i think mktime behaving different on local and on embedded is much more user hostile